extensions/UserProfile/template/en/default/pages/user_profile.html.tmpl

```diff
@@ -247,7 +247,14 @@
 <tr>
   <td>&nbsp;</td>
   <th>[% terms.Bugs %] poked</th>
-  <td class="numeric">[% stats.touched || 0 FILTER html %]</td>
+  <td class="numeric">
+  [% IF user.id %]
+      <a href="[% basepath FILTER none %]buglist.cgi?query_format=advanced&amp;emailtype1=exact&amp;emailqa_contact1=1&amp;emailassigned_to1=1&amp;emaillongdesc1=1&amp;emailbug_mentor1=1&amp;emailreporter1=1&amp;email1=[% target.login FILTER uri %]"
+        target="_blank">
```

Comment:
Usually we don't open links in new tabs/windows since the user can always right- or control-click.

Reply:
I'll fix this.